Makefile

```diff
@@ -52,6 +52,10 @@ APPLICATION_API_CRD = https://raw.githubusercontent.com/redhat-appstudio/applica
 # ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
 ENVTEST_K8S_VERSION = 1.22
 
+# Get the local platform
+OS=$(shell go env GOOS)
```

Comment:
Wouldn't`go build` be able to automatically retrieve these?

Reply:
Good question, I tried finding this exact verbage for go build on docs but didnt find any https://go.dev/doc/tutorial/compile-install

I tried various random posts and stackoverflow but didnt really find any info about it! 🤔